pages/[user].tsx

```diff
@@ -55,13 +56,52 @@ export default function User(props): User {
 }
 
 export const getServerSideProps: GetServerSideProps = async (context) => {
-  const user = await whereAndSelect(
-    prisma.user.findFirst,
-    {
-      username: context.query.user.toLowerCase(),
-    },
-    ["id", "username", "email", "name", "bio", "avatar", "theme"]
-  );
+  const team = await getTeam(context);
+  let user;
+
+  if (team) {
+    user = await prisma.user.findFirst({
+      where: {
+        username: context.query.user.toLowerCase(),
+        teams: {
```

Comment:
$where is given 1:1 to whereAndSelect - so this would have worked out of the box.

Reply:
Function prototype:
`modelQuery, criteria: Record<string, unknown>, pluckedAttributes: string[]`